nautobot_floor_plan/filter_extensions.py

```diff
@@ -19,6 +20,17 @@ class RackFilterExtension(PluginFilterExtension):
         ),
     }
 
+class UniqueRackFilterExtension(PluginFilterExtension):
+    """Add a filter to ensure a Rack is only an option if it isn't installed."""
+
+    model = "dcim.rack"
+
+    filterset_fields = {
+        "nautobot_floor_plan_unique_rack": RelatedMembershipBooleanFilter(
```

Comment:
Let's use the [core Best Practices](https://docs.nautobot.com/projects/core/en/stable/development/core/best-practices/#filter-naming-and-definition) here and use `has_floor_plan_tile` instead.